app/src/main/java/com/lumisdinos/gameoffifteen/ui/fragment/HomeFragment.kt

```diff
@@ -25,11 +25,11 @@ class HomeFragment : DaggerFragment() {
         savedInstanceState: Bundle?
     ): View? {
         viewBinding = FragmentHomeBinding.inflate(inflater, container, false)
-        val view = viewBinding.root
+        val view = viewBinding?.root
         setHasOptionsMenu(true)
-        viewBinding.root.post {
+        viewBinding?.root?.post {
             viewModel.initialLoadCells(
-                view.width,
+                view!!.width,
```

Comment:
try don't use not-null assertion , find another solution with safe call or use another construction here

Reply:
view?.width.or(0) for example

Reply:
done